tools/run_tests/run_tests.py

```diff
@@ -727,13 +727,17 @@ def test_specs(self):
                 self.args.iomgr_platform]) as tests_json_file:
             tests_json = json.load(tests_json_file)
         environment = dict(_FORCE_ENVIRON_FOR_WRAPPERS)
+        # NOTE(lidiz) Fork handlers is not designed for non-native IO manager.
+        # It has a side-effect that overrides threading settings in C-Core.
+        if args.iomgr_platform != 'native':
```

Comment:
Please file a Github issue and add a TODO in the code. Feel free to assign me to the issue.

Reply:
Filed https://github.com/grpc/grpc/issues/21401, and assigned to you.